components/configstores/nacos/config.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2021 Layotto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package nacos
+
+// map keys
+const (
+	namespaceIdKey = "namespace_id"
```

Comment:
The configuration in the metadata needs a document to explain it. Through the form of a table, let the user know what the configured key and the corresponding meaning are. The current demo only gives an example of app_name